park_info.clj

```diff
@@ -0,0 +1,367 @@
+(
+    def Bertramka{
+        ;---Basic Info---
+        :food nil
+        :food_extra nil
+        :toilet true
+        :toilet_extra nil
+        :POIs "Cultural monument, classical music concerts, social events, W.A. Mozart Museum" ;POI refers to Point of Interest
+        ;---Children and Sport---
+        :bike_path true
+        :skating false
+        :sports_field false
+        :sports_field_extra: nil
+        :playground false
+        :playground_extra: nil
+        ;---Directions---
+        :public_transport "Trams #4, #7, #9, #10, #58, #59"
+        :GPS_coords "50°4’15.00’’ N, 14°23’42.00’’ E"
+        :parking true
+        :parking_extra "OC Smíchov underground parking"
+        ;---Additional Information---
+        :dogs nil
+        :path_type "paved"
+        :administered_by "Prague 5 & Comenius Company"
+        :opening_hours "April-October: 09:00-18:00, November-March: 09:30-16:00"
+        ;---Map---
+        :map_url "https://www.praha.eu/public/fe/bd/6a/184189_4_bertramka_mapa.pdf"
+
+    }
+    def Frantiskanska_Zahrada{
+        ;---Basic Info---
+        :food true
+        :food_extra "In the vicinity - Alfa passage, Wenceslaus & Jungmann square, Narodni Trida"
+        :toilet true
+        :toilet_extra "with wheelchair access"
+        :POIs "Pleasant dining in the historical town district, suitable for both children and the elderly, Church of Our Lady of the Snows"
+        ;---Children and Sport---
+        :bike_path true
+        :skating false
+        :sports_field false
+        :sports_field_extra: nil
+        :playground true
+        :playground_extra: nil
+        ;---Directions---
+        :public_transport "Wenceslaus Sq. Trams #3, #9, #14, #24, #51, #52, #54, #55, #56, #58, metro lines: A, B station: Můstek"
```

Comment:
Maybe use a semicolon between the different transport types? As in, "...#56, #58; metro lines: A..." Make it less confusing for the end user.